internal/csi-addons/rbd/replication.go

```diff
@@ -306,6 +306,7 @@ func (rs *ReplicationServer) EnableVolumeReplication(ctx context.Context,
 
 		return nil, status.Error(codes.Internal, err.Error())
 	}
+	log.UsefulLog(ctx, "mirror state is %s", info.GetState())
```

Comment:
log rbd volume as well

Reply:
please take care of it in other places as well.

Reply:
We are already tracking the volumeID/groupID in the RPC call with this change, so I think we can skip logging the rbdVolume here.
Because, then we need to add a conditional logic for volume vs group calls